staging/src/k8s.io/client-go/INSTALL.md

```diff
@@ -83,18 +83,20 @@ go mod init
 ```
 
 Indicate which version of `client-go` your project requires.
-For `client-go` 12.0.0+, this is a single step:
+For `client-go` on master (and once version v12.0.0 is released), this is a single step:
 
 ```sh
-go get k8s.io/client-go@v12.0.0 # replace v12.0.0 with the required version
+go get k8s.io/client-go@master # or v12.0.0+ once released
 ```
 
-For `client-go` prior to v12.0.0, you also need to indicate the required versions of `k8s.io/api` and `k8s.io/apimachinery`:
+For `client-go` prior to v12.0.0, you also need to indicate the required versions of `k8s.io/api` and `k8s.io/apimachinery`
+(and any other `k8s.io/...` modules like `k8s.io/apiserver` or `k8s.io/metrics` you depend on)
```

Comment:
even many more. gnostic, kube-apiserver, gengo are very sensitive in my experience.

Reply:
I'm trying to keep this document focused on k8s.io/client-go... client-go/api/apimachinery are the only three I've need to specify to get a working build

those others might be true for apiserver, but I think the extra details should be added to a readme in those repos, not here